src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -423,6 +423,22 @@ const generateTaskRole = (frameworkName, taskRole, labels, config, storageConfig
                   name: 'STORAGE_CONFIGS',
                   value: JSON.stringify(storageConfig),
                 },
+                {
+                  name: 'PAI_TASK_ROLE_LIST',
+                  value: Object.keys(config.taskRoles).join(','),
+                },
```

Comment:
can you pass `frameworkEnvList` to init container also? then you can remove some duplicate envs